api/apps/api/test/project/request-project-export.e2e-spec.ts

```diff
@@ -42,9 +45,26 @@ test('should return bad request error if the project has a pending export', asyn
   fixtures.ThenBadRequestIsReturned(response);
 });
 
+test('should return bad request error if the project has a scenario with a pending export', async () => {
```

Comment:
(general comment) I recommend to try to make test descriptions a very brief summary of what we test from a business point of view - this can be somewhat subjective, in any case

here for example the way I would summarise this test would be "rejects a request to export a project if this has a scenario with a pending export".

again, just a general suggestion for readability, not something to specifically change here - your test description *is* correct, technically